build.sbt

```diff
@@ -184,21 +184,24 @@ lazy val libSettings =
     )
   )
 
-lazy val gcSettings =
+lazy val gcSetting =
   if (!System.getenv.containsKey("SCALANATIVE_GC")) {
-    println("Using default gc")
-    Seq.empty
+    nativeGC := {
+      val gc = nativeGC.value
+      println(s"Using $gc gc")
```

Comment:
It would be better to log using `sLog.info(s"Using $gc gc")`

https://github.com/sbt/sbt/blob/bcd21ba9b64cf89023bc9626bcb5129c155ba90b/main/src/main/scala/sbt/Keys.scala#L107

Reply:
Didn't know about that. Thank-you.